### DISCUSSION TITLE
Empty repositories

### DISCUSSION THREAD

**Discussion by NicolasGP01** [community/community#87409] [New to GitHub] [answered]
Hi, I am accessing my repositories and I find that they are totally empty, I am afraid of losing all my information.

**Comment by bonizario** [comment]


**Reply by NicolasGP01** [reply]
Hello, thank you very much for your recomendacones, actually when I access my repositories and are not, only happens to me with this team, because when entering from my cell phone and even from incognito if the files appear to me.

**Reply by NicolasGP01** [reply, marked as answer]
I fixed it, it was just a matter of clearing the browser history.

**Reply by queenofcorgis** [reply]
@bonizario,

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by 8-6-8-4** [comment]
Css ben istanbul 05352524758 arayin
